data/meta.yaml

```diff
@@ -0,0 +1,8 @@
+BaseURL: "http://example.org"
```

Comment:
Is this a duplication of `baseURL` in `config.toml`? Could be confusing to know which one to update